apps/studio/data/database-triggers/database-trigger-update-mutation.ts

```diff
@@ -18,21 +21,16 @@ export async function updateDatabaseTrigger({
   connectionString,
   payload,
 }: DatabaseTriggerUpdateVariables) {
-  let headers = new Headers()
-  if (connectionString) headers.set('x-connection-encrypted', connectionString)
+  const { sql } = pgMeta.triggers.update({ id }, payload)
```

Comment:
ditto https://github.com/supabase/postgres-meta/blob/master/src/lib/PostgresMetaTriggers.ts#L194

Reply:
Updated in https://github.com/supabase/supabase/pull/35667/commits/37c844693eb3f81dd59a5c45da60b0d4b92b8ff8